lisa/notifier.py

```diff
@@ -40,6 +42,48 @@ class TestRunMessage(MessageBase):
     message: str = ""
 
 
+@dataclass
+class PerfMessage(MessageBase):
+    pass
+
+
+class FIOResult:
```

Comment:
This class should be put with fio tool. It's fio related. Not is a part of LISA.

Reply:
updated